tests/onnx/test_pattern_manager.py

```diff
@@ -27,7 +27,6 @@
     HWFusedPatternNames.SE_BLOCK: "Not relevant for ONNX.",
     HWFusedPatternNames.SOFTMAX_DIV: "Not relevant for ONNX.",
     HWFusedPatternNames.EQUAL_LOGICALNOT: "Not relevant for ONNX.",
-    HWFusedPatternNames.HSWISH_ACTIVATION: "Not relevant for ONNX.",
     HWFusedPatternNames.HSWISH_ACTIVATION_V2: "Not relevant for ONNX.",
```

Comment:
Are you sure that `HWFusedPatternNames.HSWISH_ACTIVATION_V2` or `HWFusedPatternNames.HSWISH_ACTIVATION_WITHOUT_DENOMINATOR` are still not relevant for ONNX?

Reply:
I don't have evidence that we should add them for ONNX now

Reply:
Added `HSWISH_ACTIVATION_WITHOUT_DENOMINATOR`; add comment about `HSWISH_ACTIVATION_V2 `